Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPITestCocoa.mm

```diff
@@ -172,6 +172,9 @@
     if (!webExtensionControllerProxy)
         return;
 
+    if (m_runningTest && !m_hitAssertion)
+        m_hitAssertion = !actualValue;
```

Comment:
A simple helper for this might still be good. It can even be inline in the header.

```cpp
void recordAssertionIfNeeded(bool result)
{
    if (m_runningTest && !m_hitAssertion && !result)
        m_hitAssertion = true;
}
```